tests/plugins/test_sendmail.py

```diff
@@ -858,6 +905,7 @@ def quit(self):
             .with_args(True, False, False, False).and_return(True))
         flexmock(p).should_receive('_get_receivers_list').and_return(receivers)
         flexmock(OSBS).should_receive('get_orchestrator_build_logs').and_return(fake_logs)
+        flexmock(p).should_receive('_get_image_name_and_repos').and_return(('foobar', ['baz']))
```

Comment:
shouldn't this be returning something like : 
('foobar', ['foo/bar:baz', 'foo/bar:spam']) ??

Reply:
yeah that would be more accurate

Reply:
@rcerven I just pushed an update